Client/Frontend/Browser/BackForwardTableViewCell.swift

```diff
@@ -50,6 +50,11 @@ class BackForwardTableViewCell: UITableViewCell {
         didSet {
             if let s = site {
                 faviconView.setFavicon(forSite: s, onCompletion: { [weak self] (color, url) in
+                    if s.tileURL.isLocal {
+                        self?.faviconView.image = UIImage(named: "faviconFox" )
+                        self?.faviconView.backgroundColor = UIColor.white
```

Comment:
Missing a return here. Once this scope finishes we'll call the image setting code below.